client/my-sites/plans/current-plan/jetpack-checklist/footer.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { untrailingslashit } from 'lib/route';
+import Button from 'components/button';
+import Card from 'components/card';
+
+class JetpackChecklistFooter extends PureComponent {
+	handleWPAdminLink = () => {
+		this.props.recordTracksEvent( 'calypso_checklist_wpadmin' );
```

Comment:
Noooo idea how should I name this event.

Reply:
PCYsg-4S2-p2 is a good reference

Reply:
https://github.com/Automattic/wp-calypso/pull/32874/commits/4249d13b62d7868073fc27a64f611a92fb8101fb Renamed and added the details that are also used in the task tracking event:

https://github.com/Automattic/wp-calypso/blob/c5a6d7b5ba16a539994002778dbdc76dcdfbc726/client/my-sites/plans/current-plan/jetpack-checklist/index.js#L48-L52

https://github.com/Automattic/wp-calypso/blob/4249d13b62d7868073fc27a64f611a92fb8101fb/client/my-sites/plans/current-plan/jetpack-checklist/footer.js#L20-L23